zebrad/src/commands/start/sync.rs

```diff
@@ -169,8 +208,9 @@ where
                         "added hashes to download set"
                     );
                 }
-                Ok(r) => tracing::error!("unexpected response {:?}", r),
-                Err(e) => tracing::error!("{:?}", e),
+                Ok(_) => unreachable!("network returned wrong response"),
+                // We ignore this error because we made multiple fanout requests.
+                Err(e) => tracing::debug!(?e),
```

Comment:
#698: this error is supposed to be handled using the request fanout as failover, so it's a transient error. Demote it to `debug`.